edward/models/dirichlet_process.py

```diff
@@ -10,8 +10,9 @@
 
 
 class DirichletProcess(RandomVariable, Distribution):
-  def __init__(self, alpha, base_cls, validate_args=False, allow_nan_stats=True,
-               name="DirichletProcess", value=None, *args, **kwargs):
+  def __init__(
```

Comment:
any reason you added a linebreak? i'd default to no break otherwise, to be consistent with other class definitions in edward